tests/acceptance/features/apiGraph/getGroup.feature

```diff
@@ -132,3 +132,8 @@ Feature: get groups and their members
       }
     }
     """
+
+
+  Scenario: admin user tries to get group information of non-existing group
+    When user "Alice" gets details of the group "non-existing" using the Graph API
```

Comment:
Previously we discussed that we're going to be separating the cases for positive and negative cases and similar changes have been done here in this PR
https://github.com/owncloud/ocis/pull/5733/files 

I think we should change this step to
```
When user "Alice" gets details of the nonexistent group using the Graph API
```

Reply:
That should not be necessary in this case. The step will send a request to `non-existing` group (e.g. `.../groups/non-existing`) and the response comes with 404